src/log-uploader.test.ts

```diff
@@ -0,0 +1,239 @@
+/* tslint:disable:no-string-literal */
```

Comment:
I like disabling this only for the testing file. I just updated volt's tslint rules to turn this off. I am thinking I should go back and do this instead.